sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleLocalDateConvertersIT.java

```diff
@@ -35,6 +41,11 @@
 
   @ClassRule public static final RuleChain CHAIN = RuleChain.outerRule(MONGODB).around(DW);
 
+  @Before
+  public void cleanCollection() {
```

Comment:
Our `MONGODB` rules has a method for this.

Reply:
Fixed